io.asgardeo.java.oidc.sdk/src/main/java/io/asgardeo/java/oidc/sdk/SSOAgentConstants.java

```diff
@@ -70,6 +70,8 @@
     public static final String HTTP_CONNECT_TIMEOUT = "httpConnectTimeout";
     public static final String HTTP_READ_TIMEOUT = "httpReadTimeout";
     public static final String HTTP_SIZE_LIMIT = "httpSizeLimit";
+    public static final String STATE_PARAM = "state";
+    public static final String ADDITIONAL_PARAMS_FOR_OAUTH2_AUTHZ_ENDPOINT = "additionalParamsForAuthorizeEndpoint";
```

Comment:
Better if we can shorten the constant name. 
suggestion: CUSTOM_PARAMS_FOR_AUTHZ_EP

Reply:
fixed in e7e548c